core/validatorapi/router_internal_test.go

```diff
@@ -1147,3 +1147,36 @@ type testBeaconAddr struct {
 func (t testBeaconAddr) Address() string {
 	return t.addr
 }
+
+func NewFakeResponse() FakeResponse {
```

Comment:
you are not testing the production code, you are testing this thing.

Suggest adding a testcase to `TestRawRouter`